crates/sqlexec/src/remote/client.rs

```diff
@@ -151,13 +151,9 @@ impl RemoteClient {
         disable_tls: bool,
     ) -> Result<Self> {
         let mut dst: ProxyDestination = dst;
-        if !disable_tls {
-            info!("set rpc destination scheme to https");
-
-            dst.dst
-                .set_scheme("https")
-                .expect("failed to upgrade scheme from http to https");
-        }
+        dst.dst
+            .set_scheme("https")
+            .expect("failed to upgrade scheme from http to https");
```

Comment:
Review: If this works, I think we don't need `dst` to be `mut` anymore and can do this higher up.